Cargo.toml

```diff
@@ -8,11 +8,28 @@ members = [
     "crates/xilem_html/web_examples/todomvc",
 ]
 
+[workspace.package]
+edition = "2021"
+homepage = "https://xilem.dev/"
```

Comment:
This isn't really right - that page isn't useful. It's a shame this is now linked from `Cargo.toml`

Reply:
The page can be updated. If you think another page is more useful for now, you could just set up a simple redirect.

Reply:
The conclusion we reached for vello is to just not set the homepage field.
I think that's also right in this case, as we have not made a "real" homepage for Xilem - that page doesn't even seem to link to the repository!